Airports/EGKK/Sids.txt

```diff
@@ -56,10 +59,6 @@ SID:EGKK:08L:SFD9W:D076C D344U D344M D344G SFD
 SID:EGKK:26L:TIGER1X:KKW04 KKS06 KKS09 KKS14 MAY KKS36 LAM
 SID:EGKK:26L:TIGER3M:D260C D284M D284I MAY TIGER LAM
 SID:EGKK:26R:TIGER3V:D262C D284M D284I MAY TIGER LAM
-SID:EGKK:26L:#TIGER2M:IWW23 MAY13 MAY9 MAY TIGER LAM
```

Comment:
@agentsquash Those changes are not visible in the PR could you check you made them in the right branch?